packages/contracts/src/abis/uniswapV2Pair.ts

```diff
@@ -1,4 +1,4 @@
-export const IUniswapV2Pair = [
+export const UNISWAP_V2_PAIR_ABI = [
```

Comment:
Can we keep the original naming here so that we don't lose the `I` prefix? Or at least add it back as `Interface` somewhere in the name of this ABI 🙏🏽

Reply:
https://github.com/shapeshift/web/pull/7687/commits/c30ad7c9d58d29b9633cda65d499dab05a77ad73 ✅

Reply:
can you explain your reasoning here @gomesalexandre? the abi is for the contract which may or may not implement interfaces, but doesn't seem relevant for abi naming and feels like it should match the contract name realistically anyways.

Reply:
Pretty much that i.e the fact that this is taken from `IUniswapV2Pair.sol`! https://github.com/Uniswap/v2-core/blob/master/contracts/interfaces/IUniswapV2Pair.sol

Reply:
I guess I am saying the abi is for https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Pair.sol which is the actual contract. `UNISWAP_V2_PAIR_ABI` is actually more accurate imo. Not a huge deal, but saw it added to the bep20 abi as well and just don't think we should have it in general.